examples/SteadyStateSimulator/IEEE14/IEEE14_BasicTestCases/IEEE14_DisconnectLine/description/IEEE14.tex

```diff
@@ -0,0 +1,112 @@
+
+\begin{figure}[H]
+  \includegraphics[width=\textwidth]{Single-line-diagram-of-IEEE-14-bus-system.png}
+  \caption{IEEE 14 bus system diagram}
+\end{figure}
+
+\subsection{Initial Conditions}
+
+The reference angle for the load flow is set at bus n°1. \\
+
+Here are the initial conditions for each generator.
+
+\begin{center}
+\begin{tabular}{|c|c|c|c|c|}
+  \hline
+  Generator & P (MW) & Q (Mvar) & U (kV) & $\Theta$ (°) \\
+  \hline
+  1 & 232.39 & -16.55 & 73.14 & 0.00\\
+  2 & 40.00 & 43.56 & 72.11 & -4.98\\
+  3 & 0.00 & 25.07 & 69.69 & -12.73\\
+  6 & 0.00 & 12.73 & 14.77 & -14.22\\
+  8 & 0.00 & 17.62 & 15.04 & -13.36\\
+  \hline
+\end{tabular}
+\end{center}
+
+\subsection{Models}
+
+\subsubsection{Synchronous Machines}
+
+The generators are modelled using the GeneratorPVSignalN model. This model is used for the calculation of steady states after an event. All machines participate in the frequency regulation depending on their active power setpoint (although the frequency is not explicitly modelled in this model). All machines regulate the voltage at their stator.
+
+All parameters can be found in the IEEE14.par file.
+
+\subsubsection{System reference frequency handling}
+
+The model used for frequency handling is DYNModelSignalN. It calculates the power compensation needed to stabilize the system at each time step. This compensation is instantaneous.
+
+\subsubsection{Loads}
+
+Loads follow an alpha beta dynamic behaviour, with a load restoration when the voltage is between UMinPu and UMaxPU.
+
+They are directly connected to the grid (no transformer included).
+
+
+\subsection{Scenarios}
+The simulated scenario is a disconnection of line 1-5.
+
+\subsection{Solver}
+The solver used is the simplified solver SIM with hMin = 1 and hMax = 10.
+
+
+\newpage
+\section{Results}
+
+For each event, we focus on generator 1 's response.
+
+At $t=1s$, the line between buses 1 and 5 is opened at its bus 5 's extremity. 75.6 W and 3.1 Mvar of active and reactive power are initially transiting on this line (from bus 1 to bus 5).\\
+One can observe that the generator active power adapts to compensate the desequilibrium. The dynamics are linked to the load restoration dynamics.
+
+\begin{figure}[H]
+\subfigure[Active power (MW)]
+{%
+  \begin{tikzpicture}
+    \begin{axis}[height = 2in]
+        \addplot[color=blue!50]
+        table[x=time, y=GEN____1_SM_generator_PGenPu]
+        {../reference/outputs/curves/curves.csv};
+    \end{axis}
+  \end{tikzpicture}
+}
+\subfigure[Reactive power (Mvar)]
+{%
+  \begin{tikzpicture}
+    \begin{axis}[height = 2in]
+        \addplot[color=blue!50]
+        table[x=time, y=GEN____1_SM_generator_QGenPu]
+        {../reference/outputs/curves/curves.csv};
+    \end{axis}
+  \end{tikzpicture}
+}
+\caption{Generator 1 response to the disconnection}
+\end{figure}
+
```

Comment:
To do so, I would need to modify the voltage stability IEEE14 case a little bit for the curves to be superimposed. Some generators indeed use a proportional voltage regulation, while for this test case, all of them regulates the voltage directly (PI).

Reply:
Ok , it wouldn't be a problem for me to change the generator model for this purpose. Let's see if the other reviewers think it is a good idea or not ;)

Reply:
It raises a few questions in my mind: first of all, I believe our point in the steady-state approach is on the "steady-state" not the trajectory. Considering this, I would prefer a tabular view instead of a curves one for the results presentation.

In this one, we can add the results of the voltage stability test case. If we need to modify the voltage-stability case, perhaps it's better to have specific input data added here with only PI groups.

Reply:
@cossartque if too much work, open a new issue to address this later on.

Reply:
Ok, I replaced the curves with a table with the values before and after the event.
I also added a comparison folder with the input files for a similar simulation with DynaStab.